pages.zh/linux/xcowsay.md

```diff
@@ -0,0 +1,29 @@
+# xcowsay
+
+> 在您的 Linux 桌面上显示一头可爱的牛和指定的消息。
+> 牛的显示时间是固定的或则是根据文本大小计算得出的。 点击牛即马上关闭。
+> 更多信息： <https://www.doof.me.uk/xcowsay/>.
+
+- 显示一头说 “hello, world” 的牛：
+
+`xcowsay "{{hello, world}}"`
+
+- 显示一头有另一个命令的输出的牛：
+
+`ls | xcowsay`
+
+- 显示一头有指定 X 和 Y 坐标的牛：
+
+`xcowsay --at={{X}},{{Y}}`
+
+- 显示一头不同大小的牛：
```

Comment:
```suggestion
- 显示一头指定大小的牛：
```

Reply:
Same for this.

Reply:
I think the cow can only have one size, so the translation you provide may cause confusion. If you have other better options, you can replace it.

Reply:
The original sentence corresponds to the English page.